contracts/validator-manager/Native721TokenStakingManager.sol

```diff
@@ -674,12 +674,8 @@ contract Native721TokenStakingManager is
     ) internal override returns (uint64) {
         StakingManagerStorage storage $ = _getStakingManagerStorage();
 
-        if ($._uptimeKeeper != _msgSender()) {
-            revert OwnableUnauthorizedAccount(_msgSender());
-        }
-
         uint64 uptime = _validateUptime(validationID, messageIndex);
-        uint64 epoch = _getEpoch() - 1;
```

Comment:
not quite understanding this `-1`. why was it there? could you expand a little bit?